.github/workflows/release.yml

```diff
@@ -42,3 +42,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: npx semantic-release
+      - name: Update readme
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: "Update `README.md`"
```

Comment:
Does not follow conventional commits or skip CI.

Reply:
Also the actual pushing to main is missing here. It only commits it, or am I missing something?

Reply:
You should also set `branch: main` so it pushes to main. Otherwise it will default to the branch where the job is running on. Also only push the readme when the job is running on the main branch. can be done with an if statement

Reply:
what's wrong with updating the readme on the dev branch?

Reply:
Dev branch is not a problem. but other branches might be

Reply:
this whole workflow only runs on dev and main

Reply:
oh you're right. Forgot about that xD

Reply:
Removed.